### ISSUE TITLE
pyemma.util no attribute 'context'

### ISSUE THREAD

**Issue by pojeda** [markovmodel/PyEMMA#1567]
Hi,

I installed pyEMMA (2.5.11) with pip. I tried the test case described here http://emma-project.org/latest/tutorials/notebooks/00-pentapeptide-showcase.html#Feature-selection with my own Gromacs trajectory. Then, I get the error message: 

AttributeError: module 'pyemma.util' has no attribute 'contexts'

Do you have any comments on this error?

/Pedro

pip list:
Package                           Version
--------------------------------- -----------
alabaster                         0.7.12
appdirs                           1.4.4
asn1crypto                        1.4.0
asttokens                         2.0.5
astunparse                        1.6.3
atomicwrites                      1.4.0
attrs                             21.2.0
Babel                             2.9.1
backcall                          0.2.0
backports.entry-points-selectable 1.1.0
backports.functools-lru-cache     1.6.4
bcrypt                            3.2.0
bhmm                              0.6.3
bitstring                         3.1.9
blist                             1.3.6
CacheControl                      0.12.6
cachy                             0.3.0
certifi                           2021.5.30
cffi                              1.14.6
chardet                           4.0.0
charset-normalizer                2.0.4
cleo                              0.8.1
click                             8.0.1
clikit                            0.6.2
colorama                          0.4.4
crashtest                         0.3.1
cryptography                      3.4.7
cycler                            0.11.0
Cython                            0.29.24
decorator                         5.0.9
deeptime                          0.4.2
dill                              0.3.5.1
distlib                           0.3.2
docopt                            0.6.2
docutils                          0.17.1
ecdsa                             0.17.0
executing                         0.8.3
filelock                          3.0.12
flit                              3.3.0
flit_core                         3.3.0
fonttools                         4.33.3
fsspec                            2021.7.0
future                            0.18.2
glob2                             0.7
h5py                              3.7.0
html5lib                          1.1
humanfriendly                     10.0
idna                              3.2
imagesize                         1.2.0
importlib-metadata                4.6.3
importlib-resources               5.2.2
iniconfig                         1.1.1
intervaltree                      3.1.0
intreehooks                       1.0
ipaddress                         1.0.23
ipython                           8.4.0
jedi                              0.18.1
jeepney                           0.7.1
Jinja2                            3.0.1
joblib                            1.0.1
jsonschema                        3.2.0
keyring                           21.2.0
keyrings.alt                      4.1.0
kiwisolver                        1.4.2
liac-arff                         2.5.0
lockfile                          0.12.2
MarkupSafe                        2.0.1
matplotlib                        3.5.2
matplotlib-inline                 0.1.3
mdshare                           0.4.2
mdtraj                            1.9.7
mock                              4.0.3
more-itertools                    8.8.0
msgpack                           1.0.2
msmtools                          1.2.6
multiprocess                      0.70.13
netaddr                           0.8.0
netifaces                         0.11.0
nose                              1.3.7
numpy                             1.22.4
packaging                         20.9
paramiko                          2.7.2
parso                             0.8.3
pastel                            0.2.1
pathlib2                          2.3.6
pathos                            0.2.9
paycheck                          1.0.2
pbr                               5.6.0
pexpect                           4.8.0
pickleshare                       0.7.5
Pillow                            9.1.1
pip                               22.1.2
pkginfo                           1.7.1
platformdirs                      2.2.0
pluggy                            0.13.1
poetry                            1.1.7
poetry-core                       1.0.3
pox                               0.3.1
ppft                              1.7.6.5
prompt-toolkit                    3.0.29
psutil                            5.8.0
ptyprocess                        0.7.0
pure-eval                         0.2.2
py                                1.10.0
py-expression-eval                0.3.13
pyasn1                            0.4.8
pycparser                         2.20
pycrypto                          2.6.1
pyEMMA                            2.5.11
Pygments                          2.9.0
pylev                             1.4.0
PyNaCl                            1.4.0
pyparsing                         2.4.7
pyrsistent                        0.18.0
pytest                            6.2.4
python-dateutil                   2.8.2
pytoml                            0.1.21
pytz                              2021.1
PyYAML                            6.0
regex                             2021.8.3
requests                          2.26.0
requests-toolbelt                 0.9.1
scandir                           1.10.0
scikit-learn                      1.1.1
scipy                             1.8.1
SecretStorage                     3.3.1
semantic-version                  2.8.5
setuptools                        62.1.0
setuptools-rust                   0.12.1
setuptools-scm                    6.0.1
shellingham                       1.4.0
simplegeneric                     0.8.1
simplejson                        3.17.3
six                               1.16.0
snowballstemmer                   2.1.0
sortedcontainers                  2.4.0
Sphinx                            4.1.2
sphinx-bootstrap-theme            0.7.1
sphinxcontrib-applehelp           1.0.2
sphinxcontrib-devhelp             1.0.2
sphinxcontrib-htmlhelp            2.0.0
sphinxcontrib-jsmath              1.0.1
sphinxcontrib-qthelp              1.0.3
sphinxcontrib-serializinghtml     1.1.5
sphinxcontrib-websupport          1.2.4
stack-data                        0.2.0
tabulate                          0.8.9
threadpoolctl                     2.2.0
toml                              0.10.2
tomlkit                           0.7.2
tqdm                              4.64.0
traitlets                         5.2.2.post1
typing-extensions                 3.10.0.0
ujson                             4.0.2
urllib3                           1.26.6
virtualenv                        20.7.0
wcwidth                           0.2.5
webencodings                      0.5.1
wheel                             0.37.1
xlrd                              2.0.1
zipfile36                         0.1.3
zipp                              3.5.0

**Comment by clonker** [comment]
Hi, sorry for the super late response. You can add a `import pyemma.util.contexts` to the beginning of the notebook, then it should work fine. We have to fix this in the example.

**Comment by yihengwuKP** [comment]
Thanks for posting the solution. I would still prefer if the documentation is changed accordingly so that in the future it's easier for people to run the tutorial.

**Comment by clonker** [comment]
Definitely. Unfortunately I do not have access to the hosting servers anymore. Generally this project has gone a bit stale and probably will soon be deprecated in favor of deeptime (deeptime-ml/deeptime).
